test.sh

```diff
@@ -0,0 +1,10 @@
+. ./lib.sh
+
+msg_box "It seems like the port ${1} is closed. This could
+happend when your ISP has blocked the port, or that the port
+isn't open.
+
+If you are 100% sure the port ${1} is open you can now choose
+to continue. There are no guarantees that it will work anyway
```

Comment:
Do we call this test somewhere or is it only a test for this PR?

Reply:
> Do we call this test somewhere or is it only a test for this PR?

Oh, my god! It was careless of me. That is a file I used for a test.